services/_shared/adapters/pg_install_store.py

```
@@ -230,10 +235,22 @@ def get_repo_config(install_id: int, repo_id: int) -> dict[str, Any]:
     item = _get_item(_inst_pk(install_id), _repo_sk(repo_id)) or {}
     rid = item.get("enforcement_ruleset_id")
+
+    # Start with persona flags from registry (boolean enable/blocking flags)
     cfg: dict[str, Any] = {
-        flag: bool(item.get(flag, default))
+        flag: item.get(flag, default)
         for flag, default in _DEFAULT_PERSONA_CONFIG.items()
     }
+
+    # Merge non-persona fields (like elder_voice) - fall back to defaults
+    for key, default in _DEFAULT_REPO_CONFIG.items():
```

Ruff (0.15.20)

[warning] Loop control variable `default` not used within loop body

Rename unused `default` to `_default`

(B007)